### ISSUE TITLE
Consistency in cards when media_player is off

### ISSUE THREAD

**Issue by zappasmart** [home-assistant/frontend#4904]
## The problem
When a media_player entity is off, GLANCE and ENTITIES cards show a state badge with media content, while MEDIA CONTROL card does not show any media content picture.
It's a consistency issue.

![image](https://user-images.githubusercontent.com/53600373/74668242-18750f80-51a5-11ea-8abc-ae06931321b3.png)

## Expected behavior
Expected behaviour needs to be defined and should aim at consistency in the experience.
Should the MEDIA CONTROL card display the media content picture when state is off?
Or should the GLANCE and ENTITIES cards not display the state badge picture when state is off?

## Steps to reproduce
1. Configure a media_player entity (test was made with universal media player)
2. Add a GLANCE card, an ENTITIES card and  MEDIA CONTROL card in the UI, all including the previous media_player entity
3. Play something, the turn it off
4. Observe the different cards and in particular the presence/absence of the media content pictures

## Environment
- Home Assistant release with the issue: 0.105.4
- Last working Home Assistant release (if known): N/A
- UI Type (States or Lovelace): Lovelace
- Browser and browser version: Chrome
- Operating system: Windows

## Problem-relevant configuration
```yaml
      - type: glance
        entities:
          - entity: media_player.universal
      - type: entities
        entities:
          - entity: media_player.universal
      - type: media-control
        entity: media_player.universal
```

**Comment by zsarnett** [comment]
[![Screenshot from Gyazo](https://gyazo.com/efdd0da7de69e26691ab98504ab71026/raw)](https://gyazo.com/efdd0da7de69e26691ab98504ab71026)

I am seeing this when reproducing the issue. I would say this is more of an issue for the entities rows or something not updating the entity picture correctly. But I do see consistency issues

**Comment by zsarnett** [comment]
When the entity is off it should not show any image but an Icon for the entity. As there should be no media image to display.

**Comment by zappasmart** [comment]
I agree. Showing an image if the player is off gives the impression something is currently being played.

**Comment by zsarnett** [comment]
Something I am seeing an I would assume is the issue is when the service is called to turn the media player off. The response looks like this 
```
{
  "entity_id": "media_player.living_room_tv",
  "state": "off",
  "attributes": {
    "friendly_name": "Living Room TV",
    "supported_features": 21389
  },
}
```
What you can see here is that there is no attribute for the entity picture or entity icon. I have resolved this for the media player here: #4850

**Comment by zappasmart** [comment]
Issue to be addressed on glance and entities cards as well.
From a user perspective, it's recommended the entity icon to be displayed when the player is off, instead of the picture badge.
